config/routes.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+ActualDbSchema::Engine.routes.draw do
+  root to: "migrations#index"
+  resources :migrations, only: %i[index show] do
```

Comment:
Can we make the route as `/rails/migrations`?

Reply:
I'll try